hoot-core/src/main/cpp/hoot/core/algorithms/extractors/DistanceScoreExtractor.h

```diff
@@ -46,14 +46,14 @@ class DistanceScoreExtractor : public WayFeatureExtractor
 
   DistanceScoreExtractor();
 
-  virtual std::string getClassName() const { return className(); }
+  virtual std::string getClassName() const override { return className(); }
 
   virtual QString getDescription() const
   { return "Calculates metric distance between features"; }
 
 protected:
 
-  double _extract(const OsmMap& map, const ConstWayPtr& w1, const ConstWayPtr& w2) const;
+  double _extract(const OsmMap& map, const ConstWayPtr& w1, const ConstWayPtr& w2) const override;
```

Comment:
This is missing `virtual` from the base class `WayFeatureExtractor`.  This could actually be a problem that SonarQube found correctly.